spec/helpers/session.rb

```diff
@@ -0,0 +1,26 @@
+module SessionHelpers
+  def sign_in(email, password)
+    visit "/"
+    click_link "Get back to Peeping"
+    expect(current_path).to eq "/sessions/new"
+    fill_in "email", with: email
+    fill_in "password", with: password
+    click_button "Sign in"
+  end
+  def sign_up(user)
```

Comment:
Assignment Branch Condition size for sign_up is too high. [17/15]<br>Use empty lines between defs.